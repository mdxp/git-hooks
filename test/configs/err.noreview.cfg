[core]
	repositoryformatversion = 0
	filemode = true
	bare = false
	logallrefupdates = true
	ignorecase = true
[gate]
	name = gate
	notify = invalidemailaddress@abadaddress
	user-check = skip
[test]
	comment = 1234 Top Search  on Logo/Menu bar - \"Search\" Application
