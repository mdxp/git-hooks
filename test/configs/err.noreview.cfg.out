remote: ERROR: bug 1234 is not covered by a pending review[K
